### PR TITLE
Update how-to-write-docs-auth-pack.md

### PR DESCRIPTION
**Pull request by meganbradley**
updates for Docs Authoring Pack latest release

### CODE REVIEW

**Comment by meganbradley** [comment]
@BillWagner @JasonWHowell - updates to reflect the latest release. Please review.

**Comment by JasonWHowell** [comment]
@meganbradley Can we get a build to preview the page? I'm not sure if this is a manual build process or what happened.

File: Contribute/how-to-write-docs-auth-pack.md [hunk 1]
```diff
@@ -10,36 +10,40 @@ ms.date: 04/06/2018
...
+|Link to file in repo|Inserts a relative link to another file in the current repo. After selecting this option, type in the command window to filter files by name, then select the file you want. If you have previously selected text, it will become the link text. Otherwise, the H1 of the target file will be used as link text.|
+|Link to web page    |Inserts a link to a web page. After selecting this option, paste or type the URI into the command window. `https://` is required. If you have previously selected text, it will become the link text. Otherwise, the URI will be used as link text.|
+|Link too heading    |Links to a bookmark in the current file or another file in the repo.<br>`Bookmark in this file`: Choose from a list of headings in the current file to insert a properly formatted bookmark.<br>`Bookmark in another file`: First, filter by file name and select the file to link to, then choose the appropriate heading within the selected file.|
```
Comment (JasonWHowell) [inline Contribute/how-to-write-docs-auth-pack.md]: Typo: too

File: Contribute/how-to-write-docs-auth-pack.md [hunk 2]
```diff
@@ -68,17 +72,40 @@ To access the Docs Markdown menu, type `ALT+M`. You can click or use up/down arr
...
 
-## Known issues
+## How to use Docs templates
+
+The Docs Article Templates extension lets writers in VS Code pull a Markdown template from a centralized store and apply it to a file. Templates can help ensure that required metadata is included in articles, that content standards are followed, and so on. Templates are managed as Markdown files in a public GitHub repository.
+
+### To apply a template in VS Code
+
+1. Ensure the Docs Article Templates extension is installed and enabled.
+1. If you don't have the Docs Markdown extension installed, hit F1 to open the command palette, start typing "template" to filter, then click `Docs: Template`. If you do have Docs Markdown installed, you can use either the command palette or click `Alt+M` to bring up the Docs Markdown QuickPick menu, then select `Template` from the list.
```
Comment (JasonWHowell) [inline Contribute/how-to-write-docs-auth-pack.md]: This approach does not work for me. F1 > type template. The docs one does not appear.

File: Contribute/how-to-write-docs-auth-pack.md [hunk 3]
```diff
@@ -68,17 +72,40 @@ To access the Docs Markdown menu, type `ALT+M`. You can click or use up/down arr
...
+|----------|-----|
+|author    |Your GitHub alias, if specified in your VS Code settings file.|
+|ms.author |Your Microsoft alias, if specified in your VS Code settings file. If you are not a Microsoft employee, leave this unspecified.|
+|ms.date   |The current date in the Docs-supported format, MM/DD/YYYY. Note that the date is not automatically updated if you subsequently update the file - you must update this manually to indicate the article freshness date.|
+
+### To set author and/or ms.author
+
+1. In VS Code, go to File -> Preferences -> Settings (`CTRL+Comma`).
+1. Select User Settings to change the settings for all VS Code workspaces, or Workspace Settings to change them for just the current workspace.
+1. In the Default Settings pane on the left, find Docs Article Templates Extension Configuration, click the pencil icon next to the desired setting, then click Replace in Settings.
```
Comment (JasonWHowell) [inline Contribute/how-to-write-docs-auth-pack.md]: Do I have an old version?   
```
// Microsoft alias
  "docs.templates.alias": "",

  // GitHub ID
  "docs.templates.githubid": "",
```

**Comment by meganbradley** [comment]
@JasonWHowell we don't have preview for the public repo. Made some clarifications and filed a bug on the command discrepancy.

**Comment by JasonWHowell** [comment]
@meganbradley Look good to me. Merge as you see fit :-)

**Pull request merged**